docs/loadjdbc.adoc

```diff
@@ -224,3 +228,266 @@ CREATE (t)-[:GENRE]->(g);
+├──────┼───────┼──────┼───────┤
+│Detail│address│false │CREATED│
+├──────┼───────┼──────┼───────┤
+│Detail│salary │false │CREATED│
+├──────┼───────┼──────┼───────┤
+│Detail│id     │true  │CREATED│
+└──────┴───────┴──────┴───────┘
+----
+
+.Create Nodes in Neo4j
+[source,cypher]
+----
+CALL apoc.load.jdbc('jdbc:postgresql://localhost:5432/testdb?user=postgres&password=postgres','employee_details') yield row
+CREATE (t:Detail {id:toString(row.id), name:row.name, age:toString(row.age), address:row.address, salary:toString(row.salary)});
+----
+
+----
+Added 50 labels, created 50 nodes, set 250 properties, statement executed in 249 ms.
+----
+
+== Oracle Example
+
+== Describe EMPLOYEE_DETAILS;
+
+[source,sql]
+
+----
+Describe Employee_Details
+
+SQL> Describe EMPLOYEE_DETAILS;
+
+
++----------+--------+----------------+
+| Name     | Null?  | Type           |
++----------+--------+----------------+
+| id       |        | NUMBER         |
+| name     |        | VARCHAR(50)    |
+| age      |        | NUMBER(38)     |
+| address  |        | VARCHAR(50)    |
+| salary   |        | NUMBER(5)      |
++----------+---------------+---------+
+
+select count(*) from EMPLOYEE_DETAILS;
++----------+
+| count(*) |
++----------+
+|       50 |
++----------+
+
+----
+
+
+Download the http://www.oracle.com/technetwork/database/features/jdbc/index.html[Oracle JDBC Driver], and put it into your `$NEO4J_HOME/plugins` directory.
+
+Add this config option to `$NEO4J_HOME/conf/neo4j.conf` to make it easier to interact with the oracle instance.
+
+.Add to conf/neo4j.conf
+----
+apoc.jdbc.oracle_url.url=jdbc:oracle:thin:user/password@localhost:1521/XE
+----
+
+Restart the server.
+
+Now inspect the data with oracle database.
+
+.Count rows in products table
+[source,cypher]
+----
+CALL apoc.load.jdbc('oracle_url','employee_details') YIELD row
+RETURN count(*);
+----
+
+----
++----------+
+| count(*) |
++----------+
+|       50 |
++----------+
+----
+[source,cypher]
+----
+CALL apoc.load.jdbc('oracle_url',"employee_details") YIELD row
+RETURN row LIMIT 5;
+----
+----
+╒══════════════════════════════╕
+│row                           │
+╞══════════════════════════════╡
+│{ADDRESS: NEW DELHI, ID: 1, NA│
+│ME: ANURAG, AGE: 23, SALARY: 2│
+│5000}                         │
+├──────────────────────────────┤
+│{ADDRESS: NEW DELHI, ID: 2, NA│
+│ME: SHIVANSH SRIVASTAVA, AGE: │
+│22, SALARY: 19000}            │
+├──────────────────────────────┤
+│{ADDRESS: JODHPUR, ID: 3, NAME│
+│: SANDY, AGE: 26, SALARY: 2100│
+│0}                            │
+├──────────────────────────────┤
+│{ADDRESS: NEW DELHI, ID: 4, NA│
+│ME: MANISH, AGE: 26, SALARY: 2│
+│0000}                         │
+├──────────────────────────────┤
+│{ADDRESS: NEW DELHI, ID: 5, NA│
+│ME: HARMEET, AGE: 26, SALARY: │
+│21000}                        │
+└──────────────────────────────┘
+----
+
+Let’s create some indexes and constraints, note that other indexes and constraints will be dropped by this.
```

Comment:
I don't think, we have to do the index / constraint for all the examples, the docs should be focused on one procedure at a time. We can ink to `apoc.schema.assert` though.